### PR TITLE
Add "custom_translate_workflow" flag for gce vm image import cli tool

### PR DESCRIPTION
**Pull request by dntczdx**
Add "custom_translate_workflow" flag for gce vm image import cli tool

Background: gcloud will use gce_vm_image_import cli tool instead of daisy. "custom_translate_workflow" is a flag that we want to support to keep backward compatible.

### CODE REVIEW

**Comment by dntczdx** [comment]
Adjusted format as hint

File: cli_tools/gce_vm_image_import/main.go [hunk 1]
```diff
@@ -108,6 +109,10 @@ func validateAndParseFlags() error {
 		return fmt.Errorf("either -data_disk or -os has to be specified, but not both")
 	}
 
+	if *osID == "" && *customTranWorkflow != "" {
+		return fmt.Errorf("-custom_translate_workflow can't be specified when -osID is not specified")
```
Comment (adjackura) [inline cli_tools/gce_vm_image_import/main.go]: it's just -os

File: compute-image-tools.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adjackura) [inline compute-image-tools.iml]: You probably didn't mean to include this file

**Comment by dntczdx** [comment]
Addressed

**Pull request merged**